tests/testthat/test-inventory_table.R

```diff
@@ -0,0 +1,69 @@
+library(RInstatClimatic)
```

Comment:
```suggestion
library(cdms.products)
```
Note that build is currently failing due to this change of package name.